common4j/src/main/com/microsoft/identity/common/java/broker/IBrokerPerformanceMetricsProvider.kt

```diff
@@ -0,0 +1,5 @@
+package com.microsoft.identity.common.java.broker
+
+interface IBrokerPerformanceMetricsProvider {
+    var brokerPerformanceMetrics: BrokerPerformanceMetrics
```

Comment:
>BrokerPerformanceMetrics [](http://example.com/codeflow?start=34&length=24)

Consider making return type BrokerPerformanceMetrics?, so this can return null.

I feel EMPTY can make caller side process the values erroneously, by considering them real, which can corrupt analysis